pkg/streams/streamsGet.go

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package streams
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/galasa-dev/cli/pkg/embedded"
+	galasaErrors "github.com/galasa-dev/cli/pkg/errors"
+	"github.com/galasa-dev/cli/pkg/galasaapi"
+	"github.com/galasa-dev/cli/pkg/spi"
+	"github.com/galasa-dev/cli/pkg/streamsformatter"
+)
+
+func GetStreams(streamName string, apiClient *galasaapi.APIClient, console spi.Console) error {
```

Comment:
No format flag value being passed in? The story mentions `galasactl streams get --name "myStream" --format yaml`? Is YAML format going to be added separately?